frontend/src/bundles/common/components/stepper/components/step-icon/step-icon.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Icon } from '~/bundles/common/components/components.js';
+import { IconName } from '~/bundles/common/icons/icons.js';
+
+type Properties = {
+    step: 'complete' | 'active' | 'incomplete';
```

Comment:
I think, we should create enum for this because it is used not only here

Reply:
done